views/header.html

```diff
@@ -11,30 +11,24 @@
             <ul class="nav navbar-nav">
             <div class="imagepositioning">
                 <li>
-                    <img src="./assets/images/LOGO.png" alt="Logo" width="100" height="35">
+                    <img src="./assets/images/LOGO.png" alt="Logo" width="100" height="35" id="logo">
                   </li>
             </div>
-            <div class="homepositioning">
-                <li class="active"><a style ="color:#252D4F" href="views/home.html">Home</a></li>
+            <div class="homepositioning" id="Home">
+                <li class="active"><a style ="color:#252D4F" href="/">Home</a></li>
             </div>
-            <div class="twopositioning">
-                <li><a style ="color:#252D4F"href="#">Jobs</a></li>
+            <div class="twopositioning" id="option1">
+                <li><a style ="color:#252D4F"href="/job-roles">Jobs</a></li>
             </div>
-            <div class="threepositioning">
+            <!-- <div class="threepositioning" id="option2">
                 <li><a style ="color:#252D4F" href="#">Projects</a></li>
             </div>
-            <div class="fourpositioning">
+            <div class="fourpositioning" id="option3">
```

Comment:
Can we rename the id's to be more informative? Think of it from a readability and testing POV - informative ID's will help with understanding